tools/asm_processor/compile.sh

```diff
@@ -12,4 +12,4 @@ temp="$(mktemp --tmpdir=tools/asm_processor/tmp)"
 tools/asm_processor/asm_processor.py "$2" --assembler "$AS" > "$temp.c" &&
 $CC "$temp.c" -c -o "$temp.o" &&
 tools/asm_processor/asm_processor.py "$2" --post-process "$temp.o" --assembler "$AS" --asm-prelude include/macros.inc
-powerpc-eabi-objcopy --remove-section .mwcats.text --remove-section .comment "$temp.o" "$1"
+$DEVKITPRO/devkitPPC/bin/powerpc-eabi-objcopy --remove-section .mwcats.text --remove-section .comment "$temp.o" "$1"
```

Comment:
This breaks the workflow using the DEVKITPPC environment variable instead of DEVKITPRO.

`$DEVKITPPC/bin/powerpc-eabi-objcopy --remove-section .mwcats.text --remove-section .comment "$temp.o" "$1"`